language/testing-infra/transactional-test-runner/src/tasks.rs

```diff
@@ -24,10 +24,15 @@ pub enum RawAddress {
 }
 
 fn parse_address_literal(s: &str) -> Result<AccountAddress> {
-    let (number, _number_format) = move_compiler::shared::parse_u128(s)
-        .map_err(|e| anyhow!("Failed to parse address. Got error: {}", e))?;
-
-    Ok(AccountAddress::new(number.to_be_bytes()))
+    let (array, _) = move_compiler::shared::parse_address(s).ok_or_else(|| {
+        anyhow!(
+            "Failed to parse address {} to AccountAddress with Length {}",
+            s,
+            AccountAddress::LENGTH
+        )
+    })?;
+
+    Ok(AccountAddress::new(array))
```

Comment:
this is good, we could have also done

move_comipler::...().map(|tuple| tuple.0)

to avoid the ? and Ok